libs/extensions/angular/.storybook/styles.css

```
@@ -0,0 +1,3 @@
+.canvas {
+    background-color: var(--kirby-background-color)!important;
+}
```

Also not important:

Somehow the linter missed putting a newline at end of file

```suggestion
}

```